### PR TITLE
app setting `USER_FILE`

### PR DESCRIPTION
**Pull request by cleemansen**

### CODE REVIEW

File: tests/testthat/test_RFunction.R [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
-
-test_data <- test_data("input3_move2.rds") #file must be move2!
+test_data <- test_data("input3_move2loc_LatLon.rds")
 
 test_that("happy path", {
-  actual <- rFunction(data = test_data, sdk = "unit test", year = 2005)
-  expect_equal(unique(lubridate::year(mt_time(actual))), 2005)
+  actual <- rFunction(data = test_data, sdk = "unit test", year = 1998)
+  # TODO: how to select the column of timestamps in a move2 object?
+  expect_equal(unique(lubridate::year(actual@timestamps)), 1998)
```
Comment (cleemansen) [inline tests/testthat/test_RFunction.R:6]: Hallo @annescharf - kannst du mir helfen diesen Test zu reparieren? 

Diese `actual@timestamps` Notation ist noch von `move1`. Wie kann ich das mit `move2` bewerkstelligen?

Fehler: `Error in `actual@timestamps`: no applicable method for `@` applied to an object of class "move2"`

Comment (andreakoelzsch) [reply tests/testthat/test_RFunction.R:6]: HI @cleemansen die letzte Zeile muss sein:
expect_equal(unique(lubridate::year(mt_time(actual))), 1998)

Comment (cleemansen) [reply tests/testthat/test_RFunction.R]: Vielen Danke, hat direkt geklappt..

**Comment by cleemansen** [comment]
Release must be keep back until sprint-rollout!

**Review by andreakoelzsch — COMMENTED** [review]
Hi Clemens, I am not sure how LOCAL_FILE and USER_FILE are now supposed to be handled. DId we not say that LOCAL_FILE should stay as is?

**Comment by cleemansen** [comment]
> Hi Clemens, I am not sure how LOCAL_FILE and USER_FILE are now supposed to be handled. DId we not say that LOCAL_FILE should stay as is?

Yes, `LOCAL_FILE` should stay as it was. Do you notice a changed behavior?

**Review by andreakoelzsch — APPROVED** [review]

**Review by andreakoelzsch — APPROVED** [review]

**Pull request merged**